engine-rest/engine-rest-openapi/src/main/templates/paths/task/{id}/complete/post.ftl

```diff
@@ -0,0 +1,69 @@
+{
+
+  <@lib.endpointInfo
+      id = "complete"
+      tag = "Task"
+      desc = "Completes a task and updates process variables." />
+
+  "parameters" : [
+
+    <@lib.parameter
+        name = "id"
+        location = "path"
+        type = "string"
+        required = true
+        last = true
+        desc = "The id of the task to complete."/>
+
+  ],
+
+  <@lib.requestBody
+      mediaType = "application/json"
+      dto = "CompleteTaskDto"
+      examples = ['"example-1": {
+                     "summary": "Request Body",
+                     "description": "POST `/task/anId/complete`",
+                     "value":     {
+                       "variables": {
+                         "aVariable": {
+                           "value": "aStringValue"
+                         },
+                         "anotherVariable": {
+                           "value": 42
+                         },
+                         "aThirdVariable": {
+                           "value": true
+                         }
+                       }
+                     }
+                   }'] />
+
+  "responses" : {
+
+    <@lib.response
+        code = "200"
+        desc = "Request successful. The response contains the process variables." />
+
+    <@lib.response
+        code = "204"
```

Comment:
please add an example

Reply:
This response doesn't return any content. I will add an example for it in the HTTP Request.

Reply:
👍